CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# dbt_iterable_source v0.8.0
+
+## API Updates
+[PR #28](https://github.com/fivetran/dbt_iterable_source/pull/28) includes updates in response to the [Aug 2023 updates](https://fivetran.com/docs/applications/iterable/changelog#august2023) for the Iterable connector.
+
+## 🚨 Breaking Changes 🚨
+For customers using the new August 2023 schema, please note the following changes:
+-  We have removed `user_device_history` as the value is marginal.
+- We now allow for custom fields to be brought through in the `event` and `user` tables
```

Comment:
Isn't this `user_history` and not `user`?

Reply:
Additionally, I see we have the `iterable_user_pass_through_columns` variable. Shouldn't this instead be `iterable_user_history_pass_through_columns` to be consistent with naming?

Reply:
updated to say `user_history`